api/par-api.json

```diff
@@ -1043,7 +1115,7 @@
         "description": "Returns an array of File Formats that match all of the filter conditions supplied. Any number of filters can be used together.\nThe date filters should be supplied as ISO-8601 date times or dates, i.e. yyyy-MM-ddThh:mm:ss.SZ or yyyy-MM-dd.\nIf no time information is supplied, then 00:00:00 in UTC will be assumed. This means that if you supply modified-after and modified-before dates of 2019-01-01 and 2019-12-31, you will not match any FileFormats modified during the day of 31st December.",
         "parameters": [
           {
-            "description": "A comma separated list of GUIDs of FileFormats. This filter matches only those File Formats that are identified by one of the listed GUIDs.",
+            "description": "A comma separated list of the Name part of the ParIdentifier of FileFormats. This filter matches only those File Formats that are identified by one of the listed Names. (n.b. this header is named GUID but uses names in v1.0)",
```

Comment:
This seems to be dangerous territory, using a parameter called `guid` to hold a list on names. better to keep `guid` for a comma-separated  GUID list and introduce another header parameter `name` for this.

Reply:
@carlwilson I suggest we merge this as is, and raise a new issue to describe changes/improvements